src/tool_util.c

```diff
@@ -125,6 +125,43 @@ struct timeval tvnow(void)
 
 #endif
 
+#if defined(_WIN32)
+
+struct timeval tvrealnow(void)
+{
+  /* UNIX EPOCH (1970-01-01) in FILETIME (1601-01-01) as 64-bit value */
+  static const curl_uint64_t EPOCH = (curl_uint64_t)116444736000000000ULL;
+  SYSTEMTIME systime;
+  FILETIME ftime; /* 100ns since 1601-01-01, as double 32-bit value */
+  curl_uint64_t time; /* 100ns since 1601-01-01, as 64-bit value */
+  struct timeval now;
+
+  GetSystemTime(&systime);
```

Comment:
If the intent is to show the user's local time then [GetLocalTime](https://learn.microsoft.com/en-us/windows/win32/api/sysinfoapi/nf-sysinfoapi-getlocaltime)?

Reply:
Yeah, thanks for the pointer. In the end, the debug function does print the local time, but in the internal api, we need the epoch seconds. Using GetLocalTime would need more rewiring. So, I'll leave it as it is.